package.json

```diff
@@ -28,7 +28,8 @@
 		"test:build": "tsx tools/checkBuild.ts",
 		"test:tables": "npx markdown-table-formatter \"docs/**/*.{md,mdx}\" --check",
 		"fix:tables": "npx markdown-table-formatter \"docs/**/*.{md,mdx}\"",
-		"decorate:docs": "node tools/doxygen/generate-social-sdk-mapping.js && node tools/doxygen/decorate-social-sdk-references.js"
+		"decorate:docs": "node tools/doxygen/generate-social-sdk-mapping.js && node tools/doxygen/decorate-social-sdk-references.js",
+		"prepare": "npx is-ci || npx husky"
```

Comment:
npx is unneessary

Reply:
oh!
i assumed it is because of i.e. `npx markdown-table-formatter`

so just `is-ci || husky`?

Reply:
yep